### DISCUSSION TITLE
Noob question about using mergerfs.fsck

### DISCUSSION THREAD

**Discussion by dma9** [trapexit/mergerfs-tools#129] [Q&A]
With apologies in advance for what is likely a dumb question, are there any resources available to provide some guidance on how to choose the appropriate fix when using mergerfs.fsck? I've used fsck before but this seems to be a bit different. For example, I've just run 

`mergerfs.fsck -v -f manual /vdata`

Where `vdata` is the mountpoint of my mergerfs drive array, comprised of 16 drives each mounted at `/mnt/data/dataX/`. The first thing that's displayed is this:

```
/vdata
  0: /mnt/data/data1/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1656588115.9045804
  1: /mnt/data/data10/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1631840805.0840979
  2: /mnt/data/data11/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1631007294.0679667
  3: /mnt/data/data12/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1629938575.985874
  4: /mnt/data/data13/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1630486153.919422
  5: /mnt/data/data14/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1629990847.4548068
  6: /mnt/data/data15/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1629940166.6147249
  7: /mnt/data/data16/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1628956263.1874344
  8: /mnt/data/data2/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1656588116.1365762
  9: /mnt/data/data3/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1656588116.4125714
  10: /mnt/data/data4/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1630035126.4052842
  11: /mnt/data/data5/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1656588116.5965683
  12: /mnt/data/data6/
   - uid:     0; gid:     0; mode:  40755; size:       4096; mtime: 1656588116.7845652
  13: /mnt/data/data7/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1656588116.9725618
  14: /mnt/data/data8/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1629940166.090735
  15: /mnt/data/data9/
   - uid:  2000; gid:  2000; mode:  40777; size:       4096; mtime: 1630719312.4092271
Which is correct?: 
```
I suppose I should perhaps just pick the newest, but to be honest am not even sure of the implications of selecting the newest and am deathly afraid of borking the entire array by making the wrong choice, particularly in this case, where it seems to be identifying an issue with the mountpoint itself (`/vdata`). I'm also not quite sure I understand the "nonroot" fix option. I've tried googling to find a guide or something to provide more details but haven't had much luck.  Any advice or suggestions (or pointers to more detailed guidance), particularly in relation to how to avoid killing my whole array, would be most appreciated.

**Comment by trapexit** [comment]
No, there is no guidance because it's completely a personal and subjective concern. Only you know what the correct values are. Who the owners are and what the perms are.

It's just ownership and permission data. You can't bork anything you can't just go and fix. There is nothing you can do here that could "kill" your setup. Worse case is you have to become root and chmod and chown them.
